features.mdx

```diff
@@ -204,3 +204,15 @@ Where the name of the Python property
 and the name provided to `feature(name=...)` differ,
 IDs are [auto-assigned](/docs/features#primary-keys) based
 on the name provided to `feature(name=...)`.
+
+## Default feature values
+For features that can't be computed, you can pass `default` to 
```

Comment:
maybe features that can't `always` be computed?

Reply:
oops, I missed this and smashed merge but I just added it